google/cloud/bigtable/internal/async_retry_op.h

```diff
@@ -0,0 +1,205 @@
+// Copyright 2018 Google LLC
```

Comment:
The builds are failing because you did not add this file to the `google/cloud/bigtable/CMakeLists.txt` file.  Once you add the file there, CMake will update the right `*.bzl` file the next time you compile.